infer/src/IR/CallFlags.ml

```diff
@@ -22,8 +22,8 @@ type t =
 
 let pp f
     ({cf_assign_last_arg; cf_injected_destructor; cf_interface; cf_is_objc_block; cf_virtual}[@warning
-                                                                                               "+9"])
-    =
+                                                                                               "+9"]
+      ) =
```

Comment:
This looks a bit weird formatting.

Reply:
Indeed I will have a look

Reply:
To be fair the formatting was broken before too so this isn't a regression.

Reply:
Yes! Definitely! But @gpetiot addressed with it incredibly quickly and nicely!